app/controllers/users/webauthn_setup_controller.rb

```diff
@@ -152,12 +152,13 @@ def process_valid_webauthn(form)
         platform_authenticator: form.platform_authenticator?,
         enabled_mfa_methods_count: mfa_user.enabled_mfa_methods_count,
       )
-      Funnel::Registration::AddMfa.call(current_user.id, 'webauthn', analytics)
       mark_user_as_fully_authenticated
       handle_remember_device
       if form.platform_authenticator?
+        Funnel::Registration::AddMfa.call(current_user.id, 'webauthn_platform', analytics)
```

Comment:
It doesn't appear that we had any test coverage previously, but it would be nice to assert in a test case that the funnel had been updated as expected.

Reply:
Ah, thanks I had presumed there was. I added a tiny check to an existing spec